src/modules/form/util/formUtil.ts

```diff
@@ -1492,9 +1493,12 @@ export const getFieldOnAssessment = (
 };
 
 export const parseOccurrencePointFormDefinition = (
-  definition: FormDefinitionFieldsFragment
+  definition: FormDefinitionFieldsFragment,
+  user?: HmisUser
```

Comment:
User can't really be undefined here right? Maybe use non-null assertion from the caller?
`parseOccurrencePointFormDefinition(form.definition, user!)` or perhaps it would be helpful to have a variation of `useAuth()` that always returns `Required<HmisAuthState>`

Reply:
Thanks good point, added non-null assertion. You're right it should always be present. Previously it was empty in storybook, but I added the decorator so it will always be present there too.